modules/adding-notebook-pod-tolerations.adoc

```diff
@@ -0,0 +1,41 @@
+:_module-type: PROCEDURE
+
+[id="adding-notebook-pod-tolerations_{context}"]
+= Adding notebook pod tolerations
+
+[role='_abstract']
+You can add a toleration to notebook pods to allow them to be scheduled on tainted nodes. This is useful if you want to dedicate specific worker nodes to running notebooks, as tainting them will prevent other workloads from running on them. 
+
+////
+For example, I might have 1-2 worker nodes that are large enough to accommodate an XL notebook server. I want to ensure when a user attempts to start an XL notebook server, it gets placed on a node with an appropriate taint to accept XL notebooks. Other smaller notebooks can share the resources across the other worker nodes that might not have a taint for an XL notebook. This can help ensure users that need to do work with large notebook sizes, the resources will be available to support them.
+////
+
```

Comment:
add this example but clean up - remove first person - change to "you" or a third person